.github/workflows/scripts/gcp-vm-startup-script.sh

```diff
@@ -11,6 +14,8 @@ sudo cat /etc/ssh/sshd_config
 echo
 echo 'Modifying config:'
 echo 'MaxStartups 500' | sudo tee --append /etc/ssh/sshd_config \
+echo 'TCPKeepAlive no' | sudo tee --append /etc/ssh/sshd_config \
```

Comment:
I don't think we want this, it will make some firewalls terminate the connection as inactive.